sqlite3.go

```diff
@@ -297,6 +297,47 @@ const (
 	/*SQLITE_RECURSIVE           = C.SQLITE_RECURSIVE*/
 )
 
+// Standard File Control Opcodes
+// See: https://www.sqlite.org/c3ref/c_fcntl_begin_atomic_write.html
+const (
```

Comment:
@mattn By referencing the actual C constants here, this won't compile for anyone linking with an older version of SQLite (via the libsqlite3 tag). For example, `SQLITE_FCNTL_SIZE_LIMIT` wasn't added until 3.27.0.

How do you want to handle this?

Reply:
Ah, I didn't consider it. @benbjohnson Sorry much. Could you please rollback changes for the constants?

Reply:
@mattn @rittneje No problem. I reverted the code back to using hardcoded integers. 👍